app/models/govuk_publishing_components/audit_components.rb

```diff
@@ -38,18 +48,16 @@ def initialize(path)
       }
     end
 
-  private
-
     def find_files(files, replace)
       files.map { |file| clean_file_name(file.gsub(replace, "")) }.sort
     end
 
     def clean_file_name(name)
-      name.tr("/_-", " ").gsub(".html.erb", "").gsub(".scss", "").gsub(".js", "").gsub("spec", "").gsub(".rb", "").strip
+      name.tr("/_-", " ").gsub(".html.erb", "").gsub(".erb", "").gsub(".scss", "").gsub(".js", "").gsub("spec", "").gsub(".rb", "").strip
     end
 
     def get_component_name_from_full_path(path)
-      path.gsub("/_", "/").gsub(@templates_full_path, "").gsub(".html.erb", "").tr('\"\'', "")
+      path.gsub("/_", "/").gsub(@templates_full_path, "").gsub(".html.erb", "").gsub(".erb", "").tr('\"\'', "")
```

Comment:
Similar to previous suggestion for readability:

```suggestion
      path
        .gsub("/_", "/")
        .gsub(@templates_full_path, "")
        .gsub(".html.erb", "")
        .gsub(".erb", "")
        .tr('\"\'', "")
```